docs/src/notation.md

```diff
@@ -0,0 +1,24 @@
+# Notation overview
+
+Since manifolds include a reasonable amount of elements and functions, the
+following list tries to keep an overview of used notation throughout `Manifolds.jl`.
+The order by terminology. Thy might be used in a plain form within the code or
+when referring to that code. This is for example the case the caligraphic symbols.
+
+Within the documented functions the utf8 symbols are used whenever possible,
+as long as that renders still in TeX within this documentation.
+
+A
+
+| Symbol | Description | Also used | Comment |
+|:--:|:-------------- |:--:|:--- |
+| ``T^*_x ℳ`` | The cotangent space at ``x`` | | |
+| ``ξ`` | A cotangent vector from ``T^*_x ℳ`` | ``ξ_1,ξ_2,…,η,𝛇`` | |
+| ``F`` | A fiber | |
+| ``⟨·,·⟩`` | inner product (in ``T^*_x ℳ``) | ``⟨·,·⟩_x, g_x(·,·)`` |
+| ``ℳ`` | A manifold | ``ℳ_1, ℳ_2,…,𝒩`` | |
+| ``𝒫_{y\gets x}X`` | parallel Transport |
+| ``x`` | A point on ℳ | ``x_1,x_2,…,y,z`` | |
+| ``T_x ℳ`` | The tangent space at ``x`` | | |
+| ``X`` | A tangent vector from ``T_x ℳ`` | ``X_1,X_2,…,Y,Z`` | sometimes with the base point in the index, ``X_x``. |
```

Comment:
This upper case seems to be a new notation convention. Are we sure we want to change it everywhere?

Reply:
That is my proposal of a notation. We should agree (from that list) for a notation an I'll unify it. Currently, we use about 3 or maybe even 4 different naming conventions/notations.